apps/frontend/src/components/auth/ForgotPassword/useForgotPassword.ts

```diff
@@ -18,7 +21,18 @@ export const useForgotPassword = () => {
     }
   }, [resetError]);
 
-  const onSubmit = async (values: ForgotPasswordFormValues) => {
+  const onSubmit = async (
+    values: ForgotPasswordFormValues,
+    { setErrors }: FormikHelpers<ForgotPasswordFormValues>,
+  ) => {
+    const score = await verifyReCaptcha('forgotPassword');
```

Comment:
You should handle the case when `verifyReCaptcha` throws an error. Preferable to do this inside the `verifyReCaptcha` function.